common/app/routes/Challenges/views/Modern/Show.jsx

```diff
@@ -24,12 +24,17 @@ const propTypes = {
 
 const mapStateToProps = createSelector(
   filesSelector,
-  files => ({
-    nameToFileKey: _.reduce(files, (map, file) => {
-      map[file.name] = file.key;
-      return map;
-    }, {})
-  })
+  files => {
+    if (Object.keys(files).length === 1) {
+      return { nameToFileKey: { Editor: files[Object.keys(files)[0]].key }};
+    }
+    return {
+      nameToFileKey: _.reduce(files, (map, file) => {
+        map['Editor'] = file.key;
```

Comment:
This makes every file `Editor`.

Reply:
Yes it does 🙈 Must have forgotten to change that back!